doc/classes/String.xml

```diff
@@ -181,6 +181,25 @@
 			<argument index="1" name="placeholder" type="String" default="&quot;{_}&quot;" />
 			<description>
 				Formats the string by replacing all occurrences of [code]placeholder[/code] with [code]values[/code].
+				[b]Note:[/b] There must be one and only one [code]_[/code] in the [code]placeholder[/code].
+				[codeblocks]
+				[gdscript]
+				# Use the defalut placeholder, and a two-dimensional array.
+				var str1 := "name will be replaced to {name}, 0 maps {0}, strength is {strength}"
+				# Then the result1 is "name will be replaced to RobotGuy, 0 maps godot, strength is 9000.91"
```

Comment:
Needs an end dot here.

Reply:
Modifications have been made according to your suggestion.